core/vm/actor/builtin/v0/payment_channel/payment_channel_actor_state.hpp

```diff
@@ -112,7 +105,18 @@ namespace fc::vm::actor::builtin::v0::payment_channel {
     Buffer extra;
     Buffer proof;
   };
-  CBOR_TUPLE(PaymentVerifyParams, extra, proof);
+  CBOR_TUPLE(PaymentVerifyParams, extra);
```

Comment:
[0.9.13 has this field](https://github.com/filecoin-project/specs-actors/blob/v0.9.13/actors/builtin/paych/paych_actor.go#L141)

Reply:
Good catch, I missed it when split the actor into versions.

Reply:
fixed